### PR TITLE
177988466-mixpanel-google-redshift documents updated

### PR DESCRIPTION
**Pull request by Adorism**
## Change description

Redshift has minor edits, mixpanel is completely updated, and Google Sheets is added with Source documentation (but not yet Destination documentation)

### Development

- [X] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/docs/integrations/grouparoo-mixpanel.mdx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      <td>Export Records to Mixpanel as User Profiles</td>
+    </tr>
+  </tablebody>
+</table>
+
+## Setup
+
+You will need a Mixpanel account and some additional information from your MixPanel account in order to sync data to Mixpanel.
+
+Navigate to Settings and choose the project you want to connect to from the Subheading called Project Settings. On the Overview Page, you can find the Project Id.
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-mixpanel.mdx]: Nitpick: 
```suggestion
Navigate to Settings and choose the project you want to connect to from the subheading called Project Settings. On the Overview Page, you can find the Project Id.
```

File: pages/docs/integrations/grouparoo-mixpanel.mdx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  <tablebody>
+    <tr>
+      <td>Option Name</td>
+      <td>Description</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Token</strong> (required)
+      </td>
+      <td>Mixpanel API Token</td>
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-mixpanel.mdx:85]: We explained how to find the Project ID and user credentials.  Should we mention where to find the API Token, too?

File: pages/docs/integrations/grouparoo-redshift.mdx [hunk 3]
```diff
@@ -246,7 +246,7 @@ This type of Source provides the ability to write custom SQL code to extract dat
 
 ## Redshift Export Record
 
-Redshift Export Record allows you to set up a Grouparoo Project that uses Redshift as a [Destination](https://www.grouparoo.com/docs/getting-started/product-concepts#destination) As with Redshift Sources, to set up ClickHouse as a Destination you will need to have the App installed and configured with connection details.
+Redshift Export Record allows you to set up a Grouparoo Project that uses Redshift as a [Destination](https://www.grouparoo.com/docs/getting-started/product-concepts#destination) As with Redshift Sources, to set up Redshift as a Destination you will need to have the App installed and configured with connection details.
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-redshift.mdx:249]: Good catch!

**Review by teallarson — APPROVED** [review]
Couple of minor things, but nothing blocking.  Nice work!

File: pages/docs/integrations/grouparoo-mixpanel.mdx (hunk 2 above)
Comment (Adorism) [reply pages/docs/integrations/grouparoo-mixpanel.mdx:85]: Good idea! I added some guidance on finding that.

**Pull request merged**